sdk/python/lib/test/automation/test_local_workspace.py

```diff
@@ -327,6 +327,7 @@ def test_stack_lifecycle_local_program(self):
         stack_name = stack_namer(project_name)
         work_dir = test_path("data", project_name)
         stack = create_stack(stack_name, work_dir=work_dir)
+        print(stack)
```

Comment:
Maybe add a note here to the effect of "`print(stack)` should not throw"

Reply:
Added an `assert` to clarify it's a part of the test.